OsmIntegrator/Startup.cs

```diff
@@ -34,6 +36,9 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+
+            services.AddScoped(typeof(IUseCase<CreateChangeFileInputDto>), typeof(CreateChangeFile));
```

Comment:
@hash Ty miałeś propozycję kiedyś na zwracanie fabryki obiektów w zależności od potrzeby. Da się to tutaj zastosować?